shared/src/ledger/storage_api/queries.rs

```diff
@@ -126,10 +126,12 @@ pub trait QueriesExt {
 
     /// Determines if it is possible to send a validator set update vote
     /// extension at the provided [`BlockHeight`] in [`SendValsetUpd`].
-    ///
-    /// This is done by checking if we are at the second block of a new epoch.
     fn can_send_validator_set_update(&self, can_send: SendValsetUpd) -> bool;
 
+    /// Check if we are at a given [`BlockHeight`] offset, `height_off`, within
+    /// the current [`Epoch`].
+    fn is_deciding_offset_within_epoch(&self, height_off: u64) -> bool;
```

Comment:
I think `height_offset` is clearer as a parameter name, it's OK that it is a bit longer